### PR TITLE
Chore/update angular to 14

### PR DESCRIPTION
**Pull request by taro04**
@yutaro-elk 

It looks like this repository has completed the update to 14.
The portal and explorer in this repository worked fine in local. 
But when I pushed, I got a CI error.

It seems that `apollo-angular`, which is the cause of the CI error, does not yet support angular14.
[stackoverflow](https://stackoverflow.com/questions/73023781/angular-14-getting-error-on-installing-apollo-angular)

A, wait until `apollo-angular` support angular 14.
B, try the following [ orverride method](https://github.com/kamilkisiela/apollo-angular/issues/1789#issuecomment-1158186134). 

Which is better?

### CODE REVIEW

**Comment by yutaro-elk** [comment]
Thanks for the suggestion.
The npm version seems to be ok, and I'd like you to try the B pattern.

**Comment by yutaro-elk** [comment]
The `ng-loading-dialog` should be fine if you run it with the `-force` option.

As for the Docker build, we wanted to disable the Github action, so let's take this opportunity to disable it.

Please also disable all Github Actions related to Docker.

File: projects/shared/jest.config.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
-require('jest-preset-angular/ngcc-jest-processor');
-
 module.exports = {
   preset: 'jest-preset-angular',
   setupFilesAfterEnv: ['<rootDir>/projects/shared/src/setup-jest.ts'],
-  testPathIgnorePatterns: ['<rootDir>/projects/shared/cypress/']
+  testPathIgnorePatterns: ['<rootDir>/node_modules/', '<rootDir>/projects/shared/cypress/'],
```
Comment (taro04) [inline projects/shared/jest.config.js:4]: From the error message of the test, 
I thought that it was necessary to ignore other than cosmosclient, so I fixed it like this in local.

```
  testPathIgnorePatterns: [
    '<rootDir>/node_modules/(?!(cosmosclient)/)',
    '<rootDir>/projects/shared/cypress/',
  ],
```
[https://jestjs.io/docs/tutorial-react-native#transformignorepatterns-customization](https://jestjs.io/docs/tutorial-react-native#transformignorepatterns-customization)

Comment (taro04) [reply projects/shared/jest.config.js:4]: However, the test still fails. 
Is this fixed `testPathIgnorePatterns` wrong?

**Comment by taro04** [comment]
@yutaro-elk 

The `npm ci --force` command is now working correctly.
But, the next test fails.

```
    SyntaxError: Cannot use import statement outside a module

      2 | import { WalletType } from '../wallets/wallet.model';
      3 | import { Injectable } from '@angular/core';
    > 4 | import cosmosclient from '@cosmos-client/core';
        | ^
      5 | import { PubKey } from '@cosmos-client/core/esm/types';
      6 | import Dexie from 'dexie';
      7 |
```

Plase, confirm the question about `jest.config.js` .

**Comment by yutaro-elk** [comment]
Adding a moduleNameMapper solves this problem.

portal ver
```js
module.exports = {
  preset: 'jest-preset-angular',
  setupFilesAfterEnv: ['<rootDir>/projects/portal/src/setup-jest.ts'],
  testPathIgnorePatterns: ['<rootDir>/node_modules/', '<rootDir>/projects/portal/cypress/'],
  moduleNameMapper: {
    '@cosmos-client/core': '<rootDir>/node_modules/@cosmos-client/core/cjs/index.js',
  },
};
```

It is like telling jest what is compiled in cjs.

**Comment by taro04** [comment]
@yutaro-elk 

Thank you for your comment.
Thanks for the solution. This didn't require babel...

**Comment by yutaro-elk** [comment]
Is it still WIP because you have other work to do?
If there is no problem with the operation at a glance, it is LGTM.

**Comment by taro04** [comment]
@yutaro-elk 
Sorry, I forgot to change the status.
Please, review. 🙏

**Review by yutaro-elk — APPROVED** [review]
Thanks for the good job. LGTM.

**Pull request merged**